lib/vehicle_factory.rb

```diff
@@ -0,0 +1,39 @@
+class VehicleFactory
+    def create_vehicles(dmv_data)
+        dmv_data.map do |data|
+            data[:vin] = data[:vin_1_10]
+            data.delete(:vin_1_10)
+            data[:year] = data[:model_year]
+            data.delete(:model_year)
+            data[:engine] = :ev
+            Vehicle.new(data)
+        end
+    end
+
+    def most_popular_makemodel(vehicles)
```

Comment:
Nice work getting to these stats! This is more of a nitpick, but it could be worth putting them in another class (VehicleStatsGenerator) or something because typically our Factory classes are *just* for making new objects.